src/framework/components/element/text-element.js

```diff
@@ -1542,7 +1542,7 @@ Object.assign(pc, function () {
         set: function (value) {
             if (this._unicodeConverter !== value) {
                 this._unicodeConverter = value;
-                this.text = this._text;
+                this._setText(this._text);
```

Comment:
I think TextElement should arguably be storing the raw unconverted text and reconverting it here using the updated unicodeConverter.